skglm/tests/test_group.py

```diff
@@ -123,5 +122,43 @@ def test_vs_celer_grouplasso(n_groups, n_features, shuffle):
     np.testing.assert_allclose(model.coef_, w, atol=1e-5)
 
 
+def test_anderson_acceleration():
+    # VAR: w = rho * w + 1 which converges to w_star = 1 / (1 - rho)
+    max_iter = 1000
+    n_features = 2
+    rho = np.array([0.5, 0.8])
+    w_star = 1 / (1 - rho)
+    X = np.diag([2, 5])
+
+    # with extrapolation
+    acc = AndersonAcceleration(K=5)
+    n_iter_acc = 0
+    w = np.ones(n_features)
+    Xw = X @ w
+    for i in range(max_iter):
+        acc.extrapolate(w, Xw)
+        w = rho * w + 1
+        Xw = X @ w
+
+        if norm(w - w_star, ord=np.inf) < 1e-9:
+            n_iter_acc = i
+            break
+
+    np.testing.assert_allclose(w, w_star)
+    np.testing.assert_allclose(Xw, X @ w_star)
+
+    # without extrapolation
+    n_iter = 0
+    w = np.ones(n_features)
+    for i in range(max_iter):
+        w = rho * w + 1
+
+        if norm(w - w_star, ord=np.inf) < 1e-9:
+            n_iter = i
+            break
+
+    np.testing.assert_array_less(n_iter_acc, n_iter)  # AA should twice faster
```

Comment:
why twice ?
in practice what do you get for n_iter_acc and n_iter?

Reply:
13 for AA and 99 for the classical approach. (I meant twice in log scale)

I cannot remember where I found such a result for VAR systems.